test/e2e/lib/pages/my-plan-page.js

```diff
@@ -25,7 +25,7 @@ export default class MyPlanPage extends AsyncBaseContainer {
 	async isSecurityPlan() {
 		return await driverHelper.isEventuallyPresentAndDisplayed(
 			this.driver,
-			By.css( '[data-e2e-product-slug="jetpack_security_daily"]' )
+			By.css( '.is-jetpack-security-v2' )
```

Comment:
Could this break with new iterations of the pricing page, @rcanepa?

Reply:
Yeah. I'm going to improve it by using a data attribute. Currently, the HTML element doesn't have the data attribute so `'[data-e2e-product-slug="jetpack_security_daily"]'` doesn't find the element.